File: scala/DPEasy1/DPEasy1.scala
import scala.io.StdIn.readLine

object DPEasy1 {
  def main(args: Array[String]) {
    val name = scala.io.StdIn.readLine("What is your name?\n")
    val age = scala.io.StdIn.readLine("What is your age?\n")
    val username = scala.io.StdIn.readLine("What is your reddit username?\n")

    println(s"your name is ${name}, you are ${age} years old, and your username is ${username}")
 }
}

File: python/easy/DP_easy_1.py
from __future__ import print_function

try:
    input = raw_input
except NameError:
    pass

TEMPL = "your name is {}, you are {} years old, and your username is {}"

if __name__ == "__main__":
    name = input("What is your name?\n")
    age = input("What is your age?\n")
    reddit_username = input("What is your reddit username?\n")
    with open("DP_easy_1.txt", 'a') as f:
        f.write("\t".join((name, age, reddit_username)) + "\n")
    print(TEMPL.format(name, age, reddit_username))
